integrationtests/self/http_test.go

```diff
@@ -122,6 +123,21 @@ var _ = Describe("HTTP tests", func() {
 		Expect(string(body)).To(Equal("Hello, World!\n"))
 	})
 
+	It("request to server with two different servrname", func() {
```

Comment:
I'd like to explicitly check that this was using the same UDP conn. What do you think of introducing an HTTP handler on the server side that outputs the remote address, and then checking that both requests came from the same address?

Reply:
I will do it right away.